### PR TITLE
Added doc annotation for atlas accesslist

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if a new command or e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by gssbzn — DISMISSED** [review]
LGTM but wait for @mongodb/docs-cloud-team review

File: internal/cli/atlas/accesslists/create.go [hunk 1]
```diff
@@ -83,6 +83,10 @@ func CreateBuilder() *cobra.Command {
 		Use:   "create <entry>",
 		Short: "Create an IP access list for your project.",
 		Args:  require.ExactArgs(1),
+		Annotations: map[string]string{
+			"args":     "entry",
+			"nameDesc": "The AWS security group ID, IP address, or CIDR address of the access list entry to create.",
```
Comment (melissamahoney-mongodb) [inline internal/cli/atlas/accesslists/create.go]: Suggestion: I'd tweak this a little bit. Starting with AWS makes it sound like its only for AWS. 

```suggestion
			"entryDesc": "The IP address, CIDR address, or AWS security group ID of the access list entry to create.",
```

File: internal/cli/atlas/accesslists/delete.go [hunk 2]
```diff
@@ -50,6 +50,10 @@ func DeleteBuilder() *cobra.Command {
 		Aliases: []string{"rm"},
 		Short:   "Delete an IP access list from your project.",
 		Args:    require.ExactArgs(1),
+		Annotations: map[string]string{
+			"args":     "entry",
+			"nameDesc": "The AWS security group ID, IP address, or CIDR address of the access list entry to delete.",
```
Comment (melissamahoney-mongodb) [inline internal/cli/atlas/accesslists/delete.go]:
```suggestion
			"entryDesc": "The IP address, CIDR address, or AWS security group ID of the access list entry to delete.",
```

File: internal/cli/atlas/accesslists/describe.go [hunk 3]
```diff
@@ -58,6 +58,10 @@ func DescribeBuilder() *cobra.Command {
 		Aliases: []string{"get"},
 		Short:   "Describe an IP access list entry.",
 		Args:    require.ExactArgs(1),
+		Annotations: map[string]string{
+			"args":     "entry",
+			"nameDesc": "The AWS security group ID, IP address, or CIDR address of the access list entry to retrieve.",
```
Comment (melissamahoney-mongodb) [inline internal/cli/atlas/accesslists/describe.go]:
```suggestion
			"entryDesc": "The IP address, CIDR address, or AWS security group ID of the access list entry to retrieve.",
```

**Review by melissamahoney-mongodb — CHANGES_REQUESTED** [review]

**Review by melissamahoney-mongodb — APPROVED** [review]

**Pull request merged**